src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -80,6 +79,16 @@ class BankAccountStep extends React.Component {
         return errors[inputKey] ? this.props.translate(this.errorTranslationKeys[inputKey]) : '';
     }
 
+    setSubStep(subStep) {
+        console.debug('setSubStep', subStep);
+        if (!subStep) {
+            // undefined seems to be ignored by Onyx
+            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {achData: {subStep: null}});
+        } else {
+            Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {achData: {subStep}});
```

Comment:
As a convention we avoid using `Onyx.merge()` outside of an actions file. I would recommend checking out the readme before continuing further. https://github.com/Expensify/App#structure-of-the-app

Reply:
You are right. I read that documentation, but I forgot about that. I'll have it more present now, thanks!